### PR TITLE
Minimal version of saved command

### PR DESCRIPTION
**Pull request by alcen**
In this commit, "saved" is implemented but it does not do anything yet. Classes like `Savings` are implemented as well, but the modifications required for `Coupon` are also not implemented yet (each `Coupon` should have a `Savings` amount when created)

### CODE REVIEW

File: src/main/java/csdev/couponstash/commons/core/StashSettings.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package csdev.couponstash.commons.core;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * This class holds the settings of the CouponStash
```
Comment (Denise-Ng) [inline src/main/java/csdev/couponstash/commons/core/StashSettings.java:7]: Cool!

File: src/main/java/csdev/couponstash/logic/commands/SavedCommand.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+     *     what you have saved in the future)
+     */
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        ObservableList<Coupon> couponsList = model.getFilteredCouponList();
+        StringBuilder moneySaved = new StringBuilder("You have saved ");
+        moneySaved.append(model.getStashSettings().getMoneySymbol());
+        PureMonetarySavings pms = new PureMonetarySavings();
+        for (Coupon c : couponsList) {
+            // SavingsConverter.convertToPure(c.getSavings());
```
Comment (Denise-Ng) [inline src/main/java/csdev/couponstash/logic/commands/SavedCommand.java:45]: Is this supposed to be commented out?

Comment (alcen) [reply src/main/java/csdev/couponstash/logic/commands/SavedCommand.java:45]: Ya, Coupons don't have Savings yet

**Comment by alcen** [comment]
Need to fix merge conflicts first

**Pull request closed without merging**